.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: Build and publish to ECR
+
+on:
+  workflow_dispatch:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "Jenkinsfile"
+      - ".git**"
```

Comment:
Would be nice if paths-ignore could read from a file like `.dockerignore` :)

Reply:
don't think this is possible, maybe ask @issyl0 lol